src/index.ts

```diff
@@ -184,6 +177,10 @@ restInterface.addIMUStatusQuery("accel-reading", () => {
     return robot.getIMU().accelerationG;
 });
 
+restInterface.addIMUStatusQuery("gyro-offset", () => {
+    return robot.getIMU().gyroOffset;
```

Comment:
Do we no longer want to show the noise range found during calibration?

Reply:
I'll add it back in. This particular status query involves what the current gyro offset is set to (which could come from configuration or calibration and might not match what last-calibration-values returns if the calibration has not been run before)